scripts/prepare_automatic_backups.py

```diff
@@ -25,12 +25,22 @@
 _BACKUP_EVENT_QUEUE_RATE = '5/s'
 _MAX_BACKUP_URL_LENGTH = 2000
 _CRON_YAML_FILE_NAME = 'cron.yaml'
+_OMITTED_MODELS = [
+    'JobModel', 'ContinuousComputationModel', 'ClassifierTrainingJobModel',
+    'TrainingJobExplorationMappingModel', 'FeedbackAnalyticsModel',
```

Comment:
I think this looks correct with one exception: let's not exclude ClassifierTrainingJobModel and TrainingJobExplorationMappingModel. The second one should definitely be included. The first is a bit more iffy but I think we should err on the side of including (it tracks which jobs are still "open" and haven't been processed by our custom VM yet). Reconstituting all this might be a pain.

Reply:
Sounds good. Done.